ScrapPackedLibrary/DataTypes/PackedMetaData.cs

```diff
@@ -53,22 +62,9 @@ public PackedFileIndexData(string p_ExternalFilePath, string p_FilePath, UInt32
             ExternalFilePath = p_ExternalFilePath;
         }
 
-        public string FilePath { get; set; }
-        public string OriginalFilePath { get; private set; }
-        public UInt32 FileSize { get; set; }
-        public UInt32 OriginalOffset { get; private set; }
-        public UInt32 Offset { get; set; }
-
-        public bool UseExternalData { get {
-                return ExternalFilePath.Length != 0;
-        } }
-
-        public string ExternalFilePath { get; set; }
+        //-------------------------------------------------------
 
-        public UInt32 IndexEntrySize {
-            get {
-                return (uint)(DATA_LENGTH_STATIC + FilePath.Length);
-            }
-        }
+        // Why this is private?

```

Comment:
because it's only used internally for the calculation of IndexEntrySize?

Reply:
There is place when it will handy (in creation of new .packed). We can make it public with private setter

Reply:
Yes, you can do that once needed.